.github/workflows/ci-vendor.yml

```diff
@@ -0,0 +1,67 @@
+name: CI - Vendor
```

Comment:
added a separate workflow for testing the n most recent versions of vendor packages so that we can be notified if they ever introduce breaking changes that break our callback handler